test/test_jit.py

```diff
@@ -14153,6 +14155,8 @@ def forward(self, x, y):
 
     # unknown builtin op
     'test_nn_fold',
+
+    'test_to_sparse'
```

Comment:
You should probably list why this is here "# jit doesn't support sparse tensors."